cli/src/wallet.rs

```diff
@@ -1,88 +1,105 @@
+            }
+            WalletCommand::SetNode { url } => {
+                let wallet = set_node_command(storage_path, snapshot_path, url).await?;
+                (Some(wallet), None)
+            }
+            WalletCommand::Sync => {
+                let wallet = sync_command(storage_path, snapshot_path).await?;
+                (Some(wallet), None)
+            }
+            WalletCommand::Mnemonic => {
+                mnemonic_command().await?;
+                return Ok((None, None));
+            }
+        }
+    } else {
+        // no command provided, i.e. `> ./wallet`
+        match (storage_path.exists(), snapshot_path.exists()) {
+            (true, true) => {
+                let password = get_password("Stronghold password", false)?;
+                let wallet = unlock_wallet(storage_path, snapshot_path, &password).await?;
+                if wallet.get_accounts().await?.is_empty() {
+                    // ask the user whether a default account should be created
+                    if get_decision("Initialize a default account?")? {
+                        let account = add_account(&wallet, None).await?;
+                        println_log_info!("Initialized default account.");
+                        (Some(wallet), Some(account))
+                    } else {
+                        (Some(wallet), None)
+                    }
+                } else if let Some(account_handle) = pick_account(&wallet).await? {
+                    let account = account_handle.alias().await;
+                    (Some(wallet), Some(account))
+                } else {
+                    (Some(wallet), None)
                 }
-                WalletCommand::ChangePassword => change_password_command(&wallet, &password).await?,
-                WalletCommand::New { alias } => account = Some(new_command(&wallet, alias).await?),
-                WalletCommand::SetNode { url } => set_node_command(&wallet, url).await?,
-                WalletCommand::Sync => sync_command(&wallet).await?,
-                // PANIC: this will never happen because these variants have already been checked.
-                WalletCommand::Init(_) | WalletCommand::Mnemonic | WalletCommand::Restore { .. } => unreachable!(),
-            };
+            }
+            (false, false) => {
+                if get_decision("Initialize a new wallet with default values?")? {
+                    let wallet = init_command(storage_path, snapshot_path, InitParameters::default()).await?;
+                    println_log_info!("Initialized wallet with default values.");
 
-            (wallet, account)
+                    // ask the user whether a default account should be created
+                    if get_decision("Initialize a default account?")? {
+                        let account = add_account(&wallet, None).await?;
+                        println_log_info!("Initialized default account.");
+                        (Some(wallet), Some(account))
```

Comment:
Once the account is created, you are forwarded to the account prompt with no instruction. I think it should automatically print the help page here.

Reply:
I tend to agree but there may be different opinions here which is why I didn't include that.

Reply:
I don't disagree, but maybe for consistency there are other places where this should be done?

Reply:
can you elaborate Thibault? Why not just print the help on first run?

Reply:
I thought it was suddenly a lot of text, maybe just say "type help for a list of commands" ?

Reply:
yeah, I thought about that as well ... also a very good option I think.